projects/user_publication/publications.py

```diff
@@ -104,7 +104,7 @@ def choose_approved_with_option():
     elif inp == "3":
         return PublicationSource.APPROVED_WITH_EMAIL
     elif inp == "4":
-        return 
+        return
     elif inp == "5":
         return PublicationSource.APPROVED_WITH_PENDING_REVIEW

```

Comment:
Come to think of it, what is this status for? If the publication hasn't been reviewed, then it hasn't been approved. So how is something approved with a pending review?

Reply:
This is just to indicate that the publication source still needs approval.
I think it is better to remove this and add another field to indicate the source needs a review.